Userland/Games/MasterWord/WordGame.cpp

```diff
@@ -83,22 +83,23 @@ void WordGame::keydown_event(GUI::KeyEvent& event)
     // If we can still add a letter and the key was alpha
```

Comment:
Perhaps mention in the commit that you also renamed the field because it now serves as a general failure?